doc/nrf/protocols/multiprotocol/index.rst

```diff
@@ -46,3 +46,60 @@ Given the nature of the dynamic multiprotocol solution, take into account the fo
   The application logic must take this into account by providing a callback for all crucial transmissions and handle the ``MAC_TRANSACTION_OVERFLOW (== 241)`` status code.
 * The most time-critical part of the Zigbee stack operation is the Base Device Behavior (BDB) commissioning process.
   Postpone time-consuming operations until the ``ZB_BDB_SIGNAL_DEVICE_FIRST_START`` or ``ZB_BDB_SIGNAL_DEVICE_REBOOT`` signal is received.
+
+.. _BT_Thread_coex:
+
+Bluetooth LE and Thread coexistence
```

Comment:
I think we have to mention somewhere that we consider Bluetooth LE central role.